install/cpack_project_config.cmake

```diff
@@ -0,0 +1,33 @@
+if(CPACK_GENERATOR MATCHES "DEB")
+    # DEB package
+    if(NOT DEBARCH)
+        execute_process(
+        COMMAND dpkg --print-architecture
+        OUTPUT_VARIABLE DEBARCH
+        OUTPUT_STRIP_TRAILING_WHITESPACE
+        )
+    endif()
+    message(STATUS "Configure DEB packaging for Linux ${DEBARCH}")
+    set(CPACK_DEBIAN_PACKAGE_MAINTAINER "ZettaScale Zenoh Team, <zenoh@zettascale.tech>")
+    set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE ${DEBARCH})
+    set(CPACK_DEB_COMPONENT_INSTALL ON)
+    set(CPACK_DEBIAN_FILE_NAME DEB-DEFAULT)
+    set(CPACK_DEBIAN_LIB_PACKAGE_NAME ${CPACK_PACKAGE_NAME}) # avoid "-lib" suffix for "lib" package
```

Comment:
I think this needs to be `lib${CPACK_PACKAGE_NAME}` so the debian package will be `libzenohc`